concepticondata/concepticon.tsv

```diff
@@ -3933,4 +3933,27 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3947	HONEYCOMB	Animals	A wax structure consisting of multiple small holes made by bees to store their honey.	Person/Thing	
 3948	SHOTGUN	Warfare and hunting	A long gun (used mainly for hunting) that shoots multiple small bullets at a time.	Person/Thing	
 3949	RED-FOOTED TORTOISE	Animals	A medium-sized species of tortoise (Chelonoidis Carbonarius), found in northern parts of South America.	Person/Thing	
-3950	LIBER (INNER BARK)	Agriculture and vegetation	The inner most layer of bark (particularly from a tree).	Person/Thing	
\ No newline at end of file
+3950	LIBER (INNER BARK)	Agriculture and vegetation	The inner most layer of bark (particularly from a tree).	Person/Thing	
+3951	PANDA	Animals	Also *giant panda*, a white-and-black mammal of the *Ailuropoda* family, native to eastern Asia, commonly referred to as a type of bear.	Person/Thing	
```

Comment:
I wouldn't use `*` since they always lead to errors with Excel/OpenOffice etc. You could write:
Also called Giant Panda, a white-and-black mammal of the Ailuropoda family, native to eastern Asia, commonly referred to as a type of bear.

Reply:
Really? Then I am sorry, as I suggested to use the `*` to avoid using `"` which also messes up the TSV files when opening with libre office.

Reply:
I thought we settled for `«` and `»` for highlighting.